ocrd/processor/segment_region/tesseract3.py

```diff
@@ -0,0 +1,33 @@
+from ocrd.model import OcrdPage
+from ocrd.processor.base import Processor
+from ocrd.utils import getLogger, mets_file_id
+from ocrd.constants import MIMETYPE_PAGE
+
+import tesserocr
```

Comment:
I do not understand why the class is called `Tesseract3...`. You are importing `tesserocr` which in turn uses the tesseract installation provided at your system (which in my case is `Tesseract4`).

Reply:
I didn't realize that, because tesserocr bindings wouldn't build for me before with the 4.00 headers. But I realize now it was probably just a mismatch of headers and pkg-config. Sure, let's rename it everywhere `tesseract3` -> `tesseract` (or `tesserocr` which would be clearer but conflict-prone, global package over local package priority change between 2/3).

Reply:
BTW: If you have an idea why this won't build on Circle (https://circleci.com/gh/kba/pyocrd/12) or Travis (https://travis-ci.org/OCR-D/pyocrd/jobs/358407881) I'd appreciate it.

Reply:
```
Get:7 http://us-central1.gce.archive.ubuntu.com/ubuntu trusty/universe amd64 libtesseract-dev amd64 3.03.02-3 [1,284 kB]
```

I see the problem now, tesserocr supports only libtesseract >= 3.04 (semver for
tesseract would be a good idea to spot stuff like this).

I'll see how I can get a newer version or a newer OS in the CI container.